src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/EditStartedApprenticeship.cshtml

```diff
@@ -140,6 +142,10 @@
                 {
                     <span class="error-message" id="error-message-StartDate">The date overlaps with existing training dates for the same apprentice. Please check the date - contact your training provider for help.</span>
                 }
+                @if (!string.IsNullOrEmpty(Model.Apprenticeship.StartDateTransfersMinDate))
+                {
+                    <span class="error-message" id="error-message-StartDate">This is an alternative message</span>
```

Comment:
A better alternative message might be Model.Apprenticeship.StartDateTransfersMinDate!